### PR TITLE
[Enhancement] Add mask for credential of storage volume

### PR DESCRIPTION
**Pull request by TszKitLo40**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: test/conf/sr.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [mysql-client]
-host =
-port =
-user =
+host = 127.0.0.1
+port = 10030
+user = root
```
Comment (imay) [inline test/conf/sr.conf:4]: please don't change this conf

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 18 / 52 (34.62%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|0|31|00.00%|[1153, 1154, 1155, 1156, 1158, 1159, 1160, 1161, 1162, 1163, 1164, 1166, 1169, 1170, 1171, 1173, 1174, 1175, 1176, 1177, 1182, 1183, 1184, 1185, 1187, 1188, 1189, 1190, 1191, 1192, 1194]|
|:large_blue_circle:|com/starrocks/storagevolume/StorageVolume.java|6|9|66.67%|[188, 189, 195]|
|:large_blue_circle:|com/starrocks/sql/ast/AlterStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/CreateStorageVolumeStmt.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 18 / 52 (34.62%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|0|31|00.00%|[1153, 1154, 1155, 1156, 1158, 1159, 1160, 1161, 1162, 1163, 1164, 1166, 1169, 1170, 1171, 1173, 1174, 1175, 1176, 1177, 1182, 1183, 1184, 1185, 1187, 1188, 1189, 1190, 1191, 1192, 1194]|
|:large_blue_circle:|com/starrocks/storagevolume/StorageVolume.java|6|9|66.67%|[188, 189, 195]|
|:large_blue_circle:|com/starrocks/sql/ast/AlterStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/CreateStorageVolumeStmt.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 39 / 52 (75.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|18|31|58.06%|[1156, 1166, 1182, 1183, 1184, 1185, 1187, 1188, 1189, 1190, 1191, 1192, 1194]|
|:large_blue_circle:|com/starrocks/sql/ast/AlterStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/CreateStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/storagevolume/StorageVolume.java|9|9|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 50 / 52 (96.15%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|29|31|93.55%|[1156, 1185]|
|:large_blue_circle:|com/starrocks/sql/ast/AlterStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/CreateStorageVolumeStmt.java|6|6|100.00%|[]|
|:large_blue_circle:|com/starrocks/storagevolume/StorageVolume.java|9|9|100.00%|[]|

**Review by kevincai — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**